packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -672,29 +664,27 @@ export function buildWebpackBrowser(
                 }
               }
             }
+
+            if (!options.watch && options.serviceWorker) {
+              for (const outputPath of outputPaths) {
+                try {
+                  await augmentAppWithServiceWorker(
+                    host,
+                    root,
+                    normalize(projectRoot),
+                    normalize(outputPath),
+                    options.baseHref || '/',
```

Comment:
This is good, but I guess this won't work in a case with inlining of multiple locales and your url structure is something along these lines.

```
https://foo.com/en
https://foo.com/fr
https://foo.com/ca
```

As the contents of the SW would be absolute. Example:
```
      hashTable: {
        '/foo/favicon.ico': '84161b857f5c547e3699ddfbffc6d8d737542e01',
        '/foo/assets/folder-asset.txt': '617f202968a6a81050aa617c2e28e1dca11ce8d4',
        '/foo/index.html': 'a5359e8e1a516683b32bbb2f9e8bf402dae4738e',
      },
```

In all fairness this won't work as well, for non SW builds because of the absence of a base href in the index file.

Reply:
Why is the base HREF set here at all actually?

Reply:
I think the use of the base href here needs some more thought in general.  However, this at least allows the service worker to be created.  Worst case right now, they would then need to adjust the generated configuration files.  Not ideal but still usable.

Reply:
I take it back, there are absolute because we are setting them as such, with the baseHref of `/`.